### DISCUSSION TITLE
2022-09-07 Weekly Meeting 72 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#291] [Weekly Meetings]
Weekly Meeting 72 is happening on Wednesday September 7st, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Review UX vote #222 
- #289 

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- any others?

### Discussions: 
- Interoperable Wearables in the Metaverse

![Fb7PLEhWIAQ7KHQ](https://user-images.githubusercontent.com/8985705/188661028-da4b3e4d-5037-4506-a8de-d520e8757100.jpg)

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
and as always... thank you @antpb for setting us up!

**Comment by mrmetaverse** [comment]
# Media 
- Going to sync with A. to get their recordings into our youtube. We want to have our playlist updated by next meeting. 
  - [ ] get A. access to youtube 
  - [ ] clean playlist for next meeting

**Comment by mrmetaverse** [comment]
# MSF Delegates 
- today was an oversight meeting for MSF 
- there will be 3 documents for submitting sub groups coming 
- 4 subgroup proposals to review 
- notes: 
  - [09/07/22 Metaverse Standards Forum: Oversight](https://github.com/omigroup/MSF-Delegates/discussions/17) `#17` 
  - [7th September 2022 - MSF Delegate Sync](https://github.com/omigroup/MSF-Delegates/discussions/16) - `#16`
- Group gathered today and met with me to discuss using github and how it might/might not work for the delegates.

**Comment by mrmetaverse** [comment]
in the template repo add: 
- base milestone 
- link to finding a time to meet

**Reply by mrmetaverse** [reply]
https://www.when2meet.com/

**Reply by mrmetaverse** [reply]
emoji voting with something like hammertime in discord 
or use the bot in omi-vote

**Reply by futuristudios** [reply]
https://hammertime.cyou/

**Comment by mrmetaverse** [comment]
# Omi-Champions 
- We want to be more transparent with best practices about how we make financial decisions. Like for tools like #289 and our existing tools (gmail, 1password, miro) 
-  Champions Notes: https://github.com/omigroup/omigroup/discussions/288

**Comment by mrmetaverse** [comment]
# Demo Time! 

![image](https://user-images.githubusercontent.com/63426722/188937549-4b6001f2-328a-4b8f-aa4a-d59ac445e840.png)

**Reply by mrmetaverse** [reply]
## Jin 
- a little history of LOOT 
  - ![image](https://user-images.githubusercontent.com/63426722/188932486-75aecf7f-6da0-4e2d-a6e9-e7b1a21fc448.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188932534-26f59db7-c75e-4941-91c1-f6490ac3a183.png)
  - Its just text, and this has left it up for people to explore the designs 
  - ![image](https://user-images.githubusercontent.com/63426722/188932639-f65ecdf9-c834-4a3b-8f40-032eef6d382d.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188932668-fdc79e87-a786-4e03-ba67-7196b4bb03f6.png)
  - They have been using this as a fun art project 
  - ![image](https://user-images.githubusercontent.com/63426722/188932795-b467cebe-e2d7-485b-9c78-e96e1bdb7620.png)
- Avatar Interop Group has a repo for this 
  - ![image](https://user-images.githubusercontent.com/63426722/188932891-8f033cfd-3340-4370-9d0b-1f481ef13f93.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188932974-471ab785-0fd7-4ed8-b53a-89b785df224d.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188932997-84399ec5-67e2-4166-8249-6d26cdf6c18e.png)
  - group photo from yesterday 
    - ![image](https://user-images.githubusercontent.com/63426722/188933101-1e63adc9-51db-4104-962e-7e4184ecdda4.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188933141-0eb4c686-a0fa-4de0-86e8-70c855ca65b7.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188933197-fd7aa2d3-b9ee-4231-8bed-62baa29b34b0.png)
  - you could even attach an XR world to your avatar. 
- next is animation interop 
- avatar interop and wearable interop are co-dependent. 
- Jin is working on a grant program to improve readmes and examples. 
- Here Jin is with their clonex and wearables. 
  - ![image](https://user-images.githubusercontent.com/63426722/188933730-9b5bd20b-0b62-4032-8066-21b5caeda27b.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188933836-90cfff75-ac10-4f4d-a2f3-3cac238131f4.png)
  - dropping into NEOs and more next 
- It's amazing what stemmed from some simple text: [link to LOOT project](https://www.lootproject.com/) (trigger warning: NFT)
  - ![image](https://user-images.githubusercontent.com/63426722/188933969-fe278790-5f6a-4d88-89bd-7856e02f5c41.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188934299-ba3853c7-52af-40f1-a85b-f75b6cf0a811.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188934476-a9f9382b-c385-4f73-b89c-80c9a9a8a5f3.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188934334-8a1cc783-4548-4142-b452-72189412ad57.png)
  - ![image](https://user-images.githubusercontent.com/63426722/188934366-a4d7781a-5728-4b54-ad0f-fb5dd6c23a32.png)

**Reply by mrmetaverse** [reply]
Used AI to generate those images from the text prompt

**Reply by mrmetaverse** [reply]
https://hackmd.io/@XR/wearables1#Metadata-NFTs

**Reply by mrmetaverse** [reply]
## Jennnital of GravityLab 
![image](https://user-images.githubusercontent.com/63426722/188935250-48ac9132-07e8-423c-b11b-c94b5b643c50.png)

**Reply by mrmetaverse** [reply]
- Exploring how we will need to optimize different 3d assets for different Metaverse platforms. 
- ![image](https://user-images.githubusercontent.com/63426722/188935395-8236775a-0ede-4b76-b2a8-6df932bb07b0.png)
- What started off as an experiment has led to a lot of work for fashion week in London, and more. 
- ![image](https://user-images.githubusercontent.com/63426722/188935489-93beb6c5-9d30-4f68-8620-3031ebbc4b04.png)
- Also working with different MOCAP data through MoveAI
- ![image](https://user-images.githubusercontent.com/63426722/188935558-60b4c1a6-7f05-4196-acc6-5a686e4de630.png)
- Readyplayerme avatars 
- ![image](https://user-images.githubusercontent.com/63426722/188935685-01eb288c-c90a-469e-a687-4f48342a5843.png)
- Participated in Somnium Space Fasion Week 
- ![image](https://user-images.githubusercontent.com/63426722/188935741-328bf014-71c9-475f-84cd-9117264c930a.png)
- Had to compress the files quite a bit to run smoothly on PCVR
- ![image](https://user-images.githubusercontent.com/63426722/188935857-f41687c3-3187-4f3e-91fc-b95ccdc0e2fc.png)
- Constantly adapting different graphic styles 
- ![image](https://user-images.githubusercontent.com/63426722/188935911-f50a0b6b-ea74-4f61-aadd-c2c9915e001b.png)
- Changing the garment from its original design to work with different degrees of fidelity of graphics, file types, requirements, etc. all involve an art of manual processes. This has required an artistic eye. This is why we are so interested in finding new tech workflows to maintain quality but improve creator experience. 
- ![image](https://user-images.githubusercontent.com/63426722/188936038-be2cd68f-169b-4dfa-bcb9-bd736468ae94.png)
- Other collaborations 
- ![image](https://user-images.githubusercontent.com/63426722/188936162-dcd23225-b1f2-42de-ba83-8453c4f4bd68.png)
- ![image](https://user-images.githubusercontent.com/63426722/188936251-864346e4-382c-4bb1-a74d-0d2d4a7dbd79.png)
- The same collection for physical
- ![image](https://user-images.githubusercontent.com/63426722/188936353-0225e023-8a17-4240-98be-41cfc24e6cb4.png)
- ![image](https://user-images.githubusercontent.com/63426722/188936479-59ce582e-f9f9-49d8-a929-cd834a880a0b.png)
- Challenges the laws of fashion, with items that otherwise wouldn't be possible (due to like... gravity) \
- ![image](https://user-images.githubusercontent.com/63426722/188936527-9ac02dcd-089c-4490-a8c5-25412c32688a.png)

**Reply by mrmetaverse** [reply]
some bonus 
![image](https://user-images.githubusercontent.com/63426722/188936768-d994fdd4-7cf7-4f43-bff2-033167e95040.png)
